bin/mksetup

```diff
@@ -68,6 +68,7 @@ if ("$?EUPS_DIR" == "1" ) then
 else
    setenv EUPS_DIR "%(eupsdir)s"
 endif
+set eupslocalpath = `echo "$EUPS_DIR" | perl -pe "s/ /-+-/"`
```

Comment:
Yes. I had sed first and then switched to perl when I realised that `mksetup` was always using perl...